tracing/src/main/java/org/axonframework/extensions/tracing/OpenTraceHandlerInterceptor.java

```diff
@@ -17,8 +32,18 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+
+/**
+ * A {@link MessageHandlerInterceptor} which maps the {@link MetaData} to the {@link SpanContext}.
+ *
+ * @author Christophe Bouhier
+ * @since 4.0
+ */
 public class OpenTraceHandlerInterceptor implements MessageHandlerInterceptor<Message<?>> {
 
+    static final String TAG_AXON_PAYLOAD_TYPE = "axon.payload.type";
```

Comment:
This is just a thought, not something to fix right now, but maybe we can move these keys to a public file so that we can make these more easily configurable at a later stage?

Reply:
Makes sense, let's make this a separate issue.